api/src/main/resources/workbench.yaml

```diff
@@ -2454,6 +2541,9 @@ definitions:
   #   3) ACCESS_CONTROL_ADMIN: This is actually basically a user admin authority, for people
   #      to perform actions on a user's enabled status and manual verification.
   #   4) FEATURED_WORKSPACE_ADMIN: Allows a user to publish workspaces
+  #   5) COMMUNICATIONS_ADMIN: Allows a user to post a Status Alert banner
```

Comment:
5 and 6 are unrelated to this PR: making educated guesses here, for completeness